driftbase/api/clients.py

```diff
@@ -227,7 +227,7 @@ def post(self, args):
         }
 
         current_app.extensions['messagebus'].publish_message(
-            'clients',
+            'client',
             {'event': 'created', 'payload': payload, 'url': resource_url}
```

Comment:
The messages now use very different payloads, URLs vs IDs. Theoretically URLs would be better if we start splitting up the apps later, but until then it's probably better to focus on keeping them similar. The old one is not really used, it was just a proof of concept, so either can be changed to match the other.